src/pages/valvonta-oikeellisuus/valvonnat.svelte

```diff
@@ -253,13 +237,20 @@
                   <!-- valvonta -->
                   <td
                     class="etp-table--td"
-                    class:font-bold={isSelf(whoami, valvonta['valvoja-id'])}
-                    class:text-primary={isSelf(whoami, valvonta['valvoja-id'])}
-                    >{formatValvoja(
-                      valvojat,
+                    class:font-bold={Valvojat.isSelf(
                       whoami,
                       valvonta['valvoja-id']
-                    )}</td>
+                    )}
+                    class:text-primary={Valvojat.isSelf(
+                      whoami,
+                      valvonta['valvoja-id']
```

Comment:
Tämä pitäisi olla 
`Maybe.fold(false, Valvojat.isSelf(whoami), valvonta['valvoja-id'])`

Reply:
tästäkin voisi laittaa oman funktion tuonne valvojat-pakettiin kun paikkoja on näin paljon eli:
`export const isWhoamiInValvonta = (whoami, valvonta) => Maybe.fold(false, Valvojat.isSelf(whoami), valvonta['valvoja-id'])`

Reply:
vähän hankala vain keksiä tuolle kuvaavaa nimeä

Reply:
Sanoisin AmIInValvonts mutta nuo kaksi isoa I kirjainta vierekkäin ei ole mukava lukea niin mennään tuolla isSelfInValvonta

Reply:
sitten olisi myös 
`amIValvojaInValvonta`